src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -217,6 +229,14 @@ export const useSwapper = () => {
     return maxAmount
   }
 
+  const supportedSwappingChain = (chainId: string) => {
```

Comment:
thanks for abstracting this - do you agree that this feels like something the swapper manager should be able to answer? e.g. `swapperManager.isSupportedChain(chainId: ChainId): boolean`

Reply:
Hmm that seems like weird logic.  I could see it returning a list of swappers, but a boolean sounds weird.  Like is it supported across all swappers?  When would we need this?